### PR TITLE
Get Game State

### PR DESCRIPTION
**Pull request by GraemeKnowles**
Add route for getting the current game state based on input client id.

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -194,6 +195,16 @@ def player_count():
                                    count=len(APP.waiting_clients))
     return jsonify(response.to_dict())
 
+@APP.route('/api/game_state', methods=['GET'])
+def game_state():
+    client_state_request = ClientGameStateRequest.from_dict(dict(request.args))
+    logging.info('Parsed request: %s', client_state_request)
+
+    client = APP.get_client(client_state_request.client_id)
+    game_id = client.game_id
```
Comment (alanaalfeche) [inline server/app.py:204]: maybe we can add an if/else statement here to say if not client.game_id, just return empty or something

Comment (GraemeKnowles) [reply server/app.py:204]: That and i also forgot the jsonify on the return value

Comment (alanaalfeche) [reply server/app.py:204]: Ok I'll wait for your fix

**Comment by alanaalfeche** [comment]
@GraemeKnowles You can probably just close this PR. I'll open one along with your changes!

**Pull request closed without merging**